src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java

```diff
@@ -0,0 +1,38 @@
+// package com.backend.webproject.managers;
+
+// import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Why is this file commented?

Reply:
I commented this beacuse I am still working in that and it was giving problems when running Jacoco's report

Reply:
Okok